charts/loki-distributed/templates/role.yaml

```diff
@@ -1,10 +1,11 @@
-{{- if .Values.rbac.pspEnabled }}
+{{- if or .Values.rbac.pspEnabled .Values.rbac.sccEnabledi true }}
```

Comment:
Please fix typo and be sure the boolean logic correctness. Or true with anything will never evaluate as false.

Reply:
fixed. Thanks for catching.